src/test/java/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorSlaveIntegrationTest.groovy

```diff
@@ -115,4 +153,62 @@ class IqPolicyEvaluatorSlaveIntegrationTest
     then: 'the return code is successful'
       jenkins.assertBuildStatusSuccess(build)
   }
+
+  def 'Should perform a pipeline build on slave with newer server version'() {
+    given: 'a jenkins project'
+      WorkflowJob project = jenkins.createProject(WorkflowJob)
+      Slave slave = jenkins.createSlave()
+      configureJenkins()
+
+    and: 'a mock IQ server stub'
+      configureIqServerMock(incrementVersion(IqPolicyEvaluatorUtil.MINIMAL_SERVER_VERSION_REQUIRED))
+
+    when: 'the build is scheduled'
+      project.definition = new CpsFlowDefinition("node ('${slave.getNodeName()}') {\n" +
+          "writeFile file: 'dummy.txt', text: 'dummy'\n" +
+          "nexusPolicyEvaluation failBuildOnNetworkError: false, iqApplication: \'app\' " +
+          ", iqStage: 'stage'\n" +
+          "}\n")
+      def build = project.scheduleBuild2(0).get()
+
+    then: 'the return code is successful'
+      jenkins.assertBuildStatusSuccess(build)
+  }
+
+  def 'Should perform a pipeline build on slave with older server version'() {
```

Comment:
Should *not* perform... Right?

Reply:
updated test definition to include 'not' in 48311ed855396c55f3465376be249d7cdd7f7947